src/models/stores/stores.ts

```diff
@@ -60,9 +37,14 @@ const requiredDocumentTypes = [PersonalDocument, PlanningDocument, ProblemDocume
 export function createStores(params?: ICreateStores): IStores {
   const user = params?.user || UserModel.create({ id: "0" });
   const appConfig = params?.appConfig || AppConfigModel.create();
+  const appMode = params?.appMode || "dev";
   const demoName = params?.demoName || appConfig.appName;
+  const classInfo = params?.class || ClassModel.create({ name: "Null Class", classHash: "" });
+  const teachers: string[] = [];
+  classInfo.users.forEach(u => (u.type === "teacher") && teachers.push(u.id));
```

Comment:
I think this might not be used here anymore. I see duplicate code in: src/models/stores/user-context-provider.ts